### ISSUE TITLE
Question: Missing LOCUS identification

### ISSUE THREAD

**Issue by ne1al** [kblin/merge-gbk-records#3]
Hello, all,
I understand that `merge-gbk-records` concatenates all gbk files into one, but is there a way to keep the `LOCUS` metadata in the new merged file? It only keeps the locus of the first gbk record in the loop and I don't want to the loci of other records.
Thanks in advance

**Comment by kblin** [comment]
That's not possible at the moment, but depending on how you want the other locus tags it will be easy to add. Would having the LOCUS id of the subsequent records on the spacers preceding these records work for you?

**Comment by ne1al** [comment]
I think your suggestion might be helpful. The thing is that I have to manually annotate those loci on a visualization tool called `Clinker` for biosynthetic gene clusters. The way` merge-gbk-records` merges the records makes me lose to which locus the original record belongs. I might give your suggestion a try.

**Comment by kblin** [comment]
I think `Clinker` supports loading multiple separate GenBank records, so I don't think you need to merge them using `merge-gbk-records`.

**Comment by ne1al** [comment]
Yes, it does. But I am trying to merge GenBank records for different gene clusters present in one sample to compare them to other samples.
